modules/ROOT/pages/define-and-deploy-the-bdm.adoc

```diff
@@ -289,6 +289,12 @@ Before deploying the business data model, you must xref:pause-and-resume-bpm-ser
 You are recommended xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database before deploying a new BDM in case of problems during the update.
```

Comment:
```suggestion
It is recommended to xref:back-up-bonita-bpm-platform.adoc[Back up the platform] and database before deploying a new BDM in case of problems during the update.
```